### DISCUSSION TITLE
How to export/encode cue.value to json

### DISCUSSION THREAD

**Discussion by subramanian0331** [cue-lang/cue#2101] [Q&A] [answered]
i know we can use yaml.encode() function to export cue.value objects to yaml []byte. but how do i export them to json ?

**Comment by mvdan** [comment, marked as answer]
To clarify, you mean you want to output a JSON which itself contains a YAML-encoded string? You can do this via `cue export` with JSON as the output format, and using `yaml.Marshal` in the CUE value: https://tip.cuelang.org/play/?id=5btTpOXBoOK#cue@export@json

**Reply by subramanian0331** [reply]
thanks, is there a way to convert the json in to jsonString and add it to another object with cue file ?

**Reply by subramanian0331** [reply]
nvm i got it thanks :)

**Reply by subramanian0331** [reply]
@mvdan can we look at the source code for the Cue Playground ?

**Reply by mvdan** [reply]
Sure, see https://github.com/cue-lang/cuelang.org/tree/master/play.

**Comment by subramanian0331** [comment]
awesome! thanks :) 
i just changed the yaml to json and it marshalled it to json for me.
import "encoding/json"

#someData: {
    yamlField1: "value1"
    yamlField2: "value2"

}

jsonField1: json.Marshal(#someData)
